packages/build/src/plugins/child/validate.js

```diff
@@ -43,18 +43,13 @@ const validateProperty = function(value, propName) {
 
 // Validate `plugin.*` event handlers
 const validateMethod = function(propName) {
-  const propNameA = propName.replace(OVERRIDE_REGEXP, '')
-
-  if (!EVENTS.includes(propNameA) && LEGACY_EVENTS[propNameA] === undefined) {
-    failBuild(`Invalid event '${propNameA}'.
+  if (!EVENTS.includes(propName) && LEGACY_EVENTS[propName] === undefined) {
+    failBuild(`Invalid event '${propName}'.
```

Comment:
is this supposed to halt the execution? Because I don't see that in `failBuild` source.

Reply:
Yes `failBuild` [throws an error](https://github.com/netlify/build/blob/b80b21f5ad57d0b43fdc020a58ba4aef57f8e541/packages/build/src/plugins/error.js#L5).
This is caught at the top-level of the [plugin's process](https://github.com/netlify/build/blob/b80b21f5ad57d0b43fdc020a58ba4aef57f8e541/packages/build/src/plugins/child/main.js#L26). This is passed to the [parent process](https://github.com/netlify/build/blob/b80b21f5ad57d0b43fdc020a58ba4aef57f8e541/packages/build/src/plugins/child/main.js#L52) which eventually make the build fail and print some nice error logs.